dotcom-rendering/src/lib/acquisitions.ts

```diff
@@ -18,7 +18,7 @@ export const addTrackingCodesToUrl = ({
 	componentId,
 	campaignCode,
 	abTest,
-	pageViewId,
+	pageViewId = '',
```

Comment:
I think it’s best to leave this be `undefined`, as [`JSON.stringify`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify#description) will simply drop this key, which is probably preferable than an invalid `pageViewId` of `''`